### ISSUE TITLE
Attention for using PM

### ISSUE THREAD

**Issue by GlitchPlayer** [PocketMine/PocketMine-MP#4145]
I recommend you guys use third-party PM like Genisys. Pocketmine is now dead. Bug fixes takes longer. No 24/7 support. They left Jenkins down without fixing it. PR never get merged. Issues are fixed by locking the conversation. Some PR's are left over a year which is the better for PM.

Genisys and other third-party PM are better and up-to-date. I know we had schools and other stuffs to do but still a Success project should be monitored properly. Like listening to your people.

**Comment by GlitchPlayer** [comment]
Message to developers

Please sometimes listen to your people before posting your conclusion for the issue or PR.

**Comment by robske110** [comment]
STOP CREATING SUCH ISSUES
PM IS DEAD SO WHAT DO YOU WANT HERE
AND THEY ARE ANNOYING, and i'll still make prs

**Comment by GlitchPlayer** [comment]
Yes, this is are annoying but Im advising people to use other PM temporarily. 0.14.1 is not supported yet. But how about upcoming 0.15.0 update? People are switching to other PM because of this. Leet switched to Genisys due to outdated PM.

**Comment by GlitchPlayer** [comment]
Yes, you can make more PR but how about creating your own PM? Your only modified PM. PR are helpful and I like your PR though. But still PM is outdated

**Comment by extremeheat** [comment]
Do you not realize that by making useless issues that have nothing to do with the actual repository you are only adding on to said issue? Use a fork if you must (this is FOSS nature) but don't come back here and derail the source project. That goes against the whole premise of open source software collaboration.

Why don't you discuss these issues with PocketMine in an appropriate place instead of on GitHub issues?

0.14.1 made changes to the protocol, and nobody (as far as I can tell) has actually bothered to implement those changes (even if those changes might be minor). I'm seeing the default reaction of "new PE update; update the server protocol version" which is quite sad.

**Comment by thelucyclub** [comment]
I recommend [Cenisys](https://github.com/iTXTech/Cenisys) OR [Nukkit](https://github.com/Nukkit/Nukkit)
but PM is dead. They are faster becuase they are on C++ and Java
